From 090199ae525a5adb2a664e1acd6da09f879396e3 Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Tue, 12 Feb 2019 10:21:50 +0800
Subject: [PATCH] skx_edac: remove unsupported parameter "edac_op_state"

Parameter "edac_op_state" was added along with the initial support for
Skylake EDAC in commit 4ec656bdf. But this parameter is unused in
skx_edac driver. It cannot change error reporting state from "NMI" to
"Poll". Only interrupt mode is supported.

Upstream kernel also has this issue. Although it is harmless, to avoid
confusion, remove useless "edac_op_state" for now.

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 drivers/edac/skx_edac.c |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/edac/skx_edac.c b/drivers/edac/skx_edac.c
index 4ba92f1..f8c7eb7b 100644
--- a/drivers/edac/skx_edac.c
+++ b/drivers/edac/skx_edac.c
@@ -1182,9 +1182,6 @@ static void __exit skx_exit(void)
 module_init(skx_init);
 module_exit(skx_exit);
 
-module_param(edac_op_state, int, 0444);
-MODULE_PARM_DESC(edac_op_state, "EDAC Error Reporting state: 0=Poll,1=NMI");
-
 MODULE_LICENSE("GPL v2");
 MODULE_AUTHOR("Tony Luck");
 MODULE_DESCRIPTION("MC Driver for Intel Skylake server processors");
-- 
1.7.9.5

